checkstyle/src/main.rs

```diff
@@ -0,0 +1,133 @@
+use regex::Regex;
+use std::fs::{self};
+// use std::{fs::{self}, io::BufRead};
+use text_io::read;
+// use std::path::Path;
+// use std::path::PathBuf;
+// use std::fs::File;
+// use std::io::prelude::*;
+
+fn get_file(filename: &str) -> String {
+    fs::read_to_string(filename).expect("Failed to read file")
+}
+
+fn split_into_lines(regex: &str) -> Vec<&str> {
+    regex.lines().collect()
+}
```

Comment:
```suggestion
fn split_into_lines(s: &str) -> Vec<&str> {
    s.lines().collect()
}
```

This function doesn't have anything to do with regex in particular, so let's name our argument appropriately. `s` is a common name for just a string, but if you can think of a better one I'm open to it.